server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -147,9 +147,10 @@ private static TransportVersion registerTransportVersion(int id, String uniqueId
     public static final TransportVersion V_8_500_019 = registerTransportVersion(8_500_019, "09bae57f-cab8-423c-aab3-c9778509ffe3");
     public static final TransportVersion V_8_500_020 = registerTransportVersion(8_500_020, "ECB42C26-B258-42E5-A835-E31AF84A76DE");
     public static final TransportVersion V_8_500_021 = registerTransportVersion(8_500_021, "102e0d84-0c08-402c-a696-935f3a3da873");
+    public static final TransportVersion V_8_500_022 = registerTransportVersion(8_500_022, "4993c724-7a81-4955-84e7-403484610091");
```

Comment:
nit . It seems weird at first glance to introduce a new version that's used elsewhere. I wonder if there's a better way, but I cannot think of something quickly now. Maybe a comment here that this is used in stateless might be useful.